android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -1259,11 +1254,6 @@
 	/**
 	 * @module.api
 	 */
-	public static final String PROPERTY_DISABLE_CONTEXT_MENU = "disableContextMenu";
```

Comment:
I think you swapped what we wanted to keep versus cut in your PR. This PR is about implementing disableContextMenu, not disableTextSelection.